addOns/paramminer/src/main/javahelp/org/zaproxy/addon/paramminer/resources/help/contents/tab.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<html>
+<head>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<title>Param Miner tab</title>
+</head>
+<body>
+	<h1>Param Miner tab</h1>
+	<p>
+		The Param Miner tab shows you the set of vulnerable parameters found by the <a
+			href="paramminer.html">Param Miner</a> during the scans.
+	</p>
+
+	<p>
+		The 'New Scan' button launches the <a href="dialog.html">Param Miner dialog</a> which allows you
+to specify exactly which URL should be scanned.<br>
+	
+		The Param Miner can be run on multiple URLs in parallel and the results for each scan are
+		shown by selecting the scan via the 'Progress' pull-down.
+	</p>
+
+	<p>The toolbar shows information about a scan and allows to control it. It provides a
+		set of buttons which allows to:
+	<ul>
+		<li><img align="bottom" alt="pause button" src="images/pause.png">
+			Pause (and <img alt="resume button" src="images/play.png"> resume) the
+			selected spider scan;</li>
+		<li><img align="bottom" alt="stop button" src="images/stop.png"> Stop
+			the selected spider scan;</li>
+		<li><img align="bottom" alt="clean scans button" src="images/broom.png">
+			Clean completed scans;</li>
+		<li><img align="bottom" alt="spider options button" src="images/gear.png">
+			Open the <a href="options.html">Spider Options screen</a>.</li>
+	</ul>
+	The progress bar shows how far the selected Param Miner scan has progressed. There are two tabs provided by the Param
+    Miner, the 'History' tab and the 'Output' tab.
+	<p>In the 'History' tab for each URL found you can see:
```

Comment:
“Parameter” found? (I haven’t looked at the UI, but the tool finds params not URLs right? Or are these like extracted GET URLs?

Reply:
These URLs are the ones which the Param Miner tries. These are there to help the users figure out what kind of requests causes "a certain param to fire".

Reply:
“URL attempted”?
Or
“message sent”?